v2/googlecloud-to-neo4j/src/main/java/com/google/cloud/teleport/v2/neo4j/actions/preload/IPreloadAction.java

```diff
@@ -0,0 +1,14 @@
+package com.google.cloud.teleport.v2.neo4j.actions.preload;
+
+import com.google.cloud.teleport.v2.neo4j.model.job.Action;
+import com.google.cloud.teleport.v2.neo4j.model.job.ActionContext;
+import java.util.List;
+
+/**
+ * Interface for running preload Actions.
+ * Before the pipeline loads, PCollections are not available.
+ */
+public interface IPreloadAction {
```

Comment:
Please remove `I` in all interface names for consistency as we don't use this naming convention anywhere in this project.

If there is a default implementation the naming convention is to name the interface "PreloadAction" and the class "DefaultPreloadAction" (instead of "IPreloadAction" interface / "PreloadAction" class).

Reply:
Done